okhttp/src/main/java/okhttp3/internal/concurrent/Task.kt

```diff
@@ -35,14 +28,26 @@ package okhttp3.internal.concurrent
  * earliest one wins. This applies to both executions scheduled with [TaskRunner.Queue.schedule] and
  * those implied by the returned execution delay.
  *
+ * Cancellation
+ * ------------
+ *
+ * Tasks may be scheduled while they are waiting to be executed, or while they are executing.
+ *
+ * Canceling a task that is waiting to execute prevents that upcoming execution. Canceling a task
+ * that is currently executing does not impact the ongoing run, but it does prevent a recurrence
+ * from being scheduled.
+ *
+ * Tasks may opt-out of cancellation with `cancelable = false`.
```

Comment:
Can you expand on this, specifically does it mean that future executions will still be run even though task queue was cancelled?